src/test/java/picard/illumina/IlluminaBasecallsToSamTest.java

```diff
@@ -176,30 +177,32 @@ public void testMolecularBarcodes(final String readStructure, final String[] umi
     @Test(dataProvider = "multiplexedData")
     public void testMultiplexed(final boolean includeBcInHeader, final ClusterDataToSamConverter.PopulateBarcode populateBarcode,
                                 final boolean includeBarcodeQuality, final File testDataDir) throws Exception {
-        runStandardTest(new int[]{1}, "multiplexedBarcode.", "library.params", 1, "25T8B25T", BASECALLS_DIR, testDataDir, null, includeBcInHeader, populateBarcode, includeBarcodeQuality);
+        runStandardTest(new int[]{1}, "multiplexedBarcode.", "library.params", 1, "25T8B25T", BASECALLS_DIR, BASECALLS_DIR, testDataDir, null, includeBcInHeader, populateBarcode, includeBarcodeQuality);
```

Comment:
Would it be a lot of work to add a separate directory for the Barcodes rather than storing them in the same directory as the basecalls? I am concerned that this would test would not catch the but that is fixed here

Reply:
There is one test that does just that. All of the other test cases use the default case of using the same directory for both.